containers/scripts/docker-compose-taskrunner.yml

```diff
@@ -44,6 +44,7 @@ services:
 #      TERARIUM_MQ_PASSWORD: "terarium123"
 #      TERARIUM_MQ_USERNAME: "terarium"
 #      TERARIUM_TASKRUNNER_REQUEST_TYPE: "mira"
+#      MIRA_REST_URL: "http://mira-epi-dkg-lb-dc1e19b273dedaa2.elb.us-east-1.amazonaws.com/"
```

Comment:
Is this being used? I can't seem to find a reference to it in the rest of your PR

Reply:
Yes, this is needed by MIRA ontological client.